Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM tomcat:8.5.65-jdk11-openjdk-slim-buster
+FROM tomcat:8.5-jdk11-openjdk-slim-buster
```

Comment:
The preference here is to know what specific version it was created with vs pulling in latest at build time.